rplugin/python3/denite/source/contextMenu.py

```diff
@@ -10,10 +18,17 @@ def __init__(self, vim):
         self.kind = 'command'
 
     def convert_to_candidate(self, item):
+        if item in DeniteOverrides:
+            cmd = 'call denite#start({})'.format(
+                repr([{"name": DeniteOverrides[item],
```

Comment:
Is this `repr` necessary here?

Reply:
For lists `str()` calls `repr()` on the underlying objects so the effect is identical. But the *intent* is to use `repr()`. If it wasn't a list, maybe a string for example, `repr()` would have been required.

This documents the fact that we *want* the `repr()` of the object and not the `str()` and ignores the fact that for lists coincidentally `str() == repr()`.

Reply:
My point here is, `repr` is not necessary, nor `str`, as the goal is formatting one string.

How about this?
```python
cmd = 'call denite#start(["name": {}, "args": []])'.format(DeniteOverrides[item])
```

Reply:
Oh, I didn't get that this is what you meant. I guess you are right.